requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx
+sphinx-rtd-theme
+git+https://github.com/OCA/openupgradelib.git@master
```

Comment:
Is this library needed?

Reply:
Yes. 
It allows to generate the description of the functions of the lib here : 
https://oca.github.io/OpenUpgrade/API.html

If we remove this requirement, the following call (in API.rst file) will fail.
``.. automodule:: openupgradelib.openupgrade``

(In fact install odoo is also required to generate the docs !)
See github actions.
https://github.com/OCA/OpenUpgrade/blob/documentation/.github/workflows/documentation.yml#L49